include/pybind11/native_enum.h

```diff
@@ -24,9 +24,10 @@ class native_enum : public detail::native_enum_data {
 
     native_enum(const object &parent_scope,
                 const char *name,
-                const char *native_type_name = "enum.Enum")
+                const char *native_type_name,
+                const char *doc = "")
```

Comment:
Drive-by comment: Could you please use the same (longer) name consistently here and in native_enum_data.h?

Reply:
done